modules/selenium/src/test/java/org/testcontainers/junit/BrowserWebDriverContainerTest.java

```diff
@@ -0,0 +1,34 @@
+package org.testcontainers.junit;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testcontainers.containers.BrowserWebDriverContainer;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+
+public class BrowserWebDriverContainerTest {
+
+    private static final String NO_PROXY_VALUE = "localhost,.noproxy-domain.com";
+
+    @Rule
```

Comment:
It's probably better to define the containers inside each tests, so we don't start 2 containers for each test, although only one is needed.

You can start them manually using `start()` method and declare them inside a try-with-resources block.

Reply:
Since the container is not (Auto)Closable I don't know how to do that in a try-with-resources block. However a regular try-finally can do it.

Reply:
> Since the container is not (Auto)Closable

It is actually `AutoCloseable` via inheritance from `GenericContainer`...